Shared/Samples/Browse building floors/BrowseBuildingFloorsView.swift

```diff
@@ -37,10 +37,16 @@ struct BrowseBuildingFloorsView: View {
         )
     )
     
+    /// The height of the attribution bar at the bottom of the map view.
+    @State private var height: CGFloat?
+    
     var body: some View {
         MapView(map: map)
             .onViewpointChanged(kind: .centerAndScale) { viewpoint = $0 }
             .onNavigatingChanged { isMapNavigating = $0 }
+            .onAttributionBarHeightChanged { height in
+                withAnimation { self.height = height }
```

Comment:
Changes look good. Can you rename this variable to `attributionBarHeight`? Thanks.
```suggestion
    @State private var attributionBarHeight: CGFloat?
    
    var body: some View {
        MapView(map: map)
            .onViewpointChanged(kind: .centerAndScale) { viewpoint = $0 }
            .onNavigatingChanged { isMapNavigating = $0 }
            .onAttributionBarHeightChanged { height in
                withAnimation { attributionBarHeight = height }
```